src/lib/ai/AIService.ts

```
@@ -566,6 +572,57 @@
+          for (const tech of documentation.therapeuticTechniques) {
+            if (tech.name) desiredOutcomes.push(tech.name)
+          }
+        }
+        // Remove duplicates and sanitize
+        const uniqueOutcomes = Array.from(new Set(desiredOutcomes.map(o => o.trim()).filter(Boolean)))
+        let outcomePredictions: SessionDocumentation['outcomePredictions'] = []
+        if (uniqueOutcomes.length > 0) {
+          try {
+            // Dynamically import OutcomeRecommendationEngine to avoid circular deps
+            // eslint-disable-next-line @typescript-eslint/no-var-requires
+            const { OutcomeRecommendationEngine } = require('./services/OutcomeRecommendationEngine')
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)